servers/republik/script/migratePortraitUrls.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+/**
+ * This script migrates the old portraitUrl format
+ *
+ * Usage:
+ * ./script/fixMemberRole.js [restartCount]
+ */
+require('@orbiting/backend-modules-env').config()
+const fetch = require('isomorphic-unfetch')
+const Promise = require('bluebird')
+const PgDb = require('@orbiting/backend-modules-base/lib/PgDb')
+const { lib: { upload: { uploadPortrait } } } = require('@orbiting/backend-modules-assets')
+
+console.log('running migratePortraitUrls.js...')
+PgDb.connect().then(async pgdb => {
+  const dry = process.argv[2] === '--dry'
+
+  let users = await pgdb.public.users.find({
+    'portraitUrl like': '%384x384%'
+  })
+  console.log(`num users to migrate: ${users.length}`)
+  let progress = 0
+
+  await Promise.map(
+    users,
+    async (user) => {
+      let { portraitUrl } = user
+      portraitUrl = portraitUrl.replace('_384x384', '_original')
+      // download from s3 directly to avoid recompression
+      portraitUrl = portraitUrl.replace('https://assets.republik.ch/', 'https://republik-assets.s3.eu-central-1.amazonaws.com/')
```

Comment:
Throws me a bit off to see this hard-coded.

But I understand this is a one-off script.